### PR TITLE
feat(Discord Node): Overhaul

### PR DESCRIPTION
**Pull request by michael-radency**
Github issue / Community forum post (link here to close automatically):

### CODE REVIEW

**Comment by paulwer** [comment]
@michael-radency 
fyi: there is a nodejs library called [discord.js](https://discord.js.org/)
maybe this helps in getting an overview, which features among the currently visible (in commit history) can be usefull.
I am thinking exspecialy of events and right-management of servers.

**Comment by luizeof** [comment]
@michael-radency I was very interested, if you can create a dockerhub image I can do tests for you.

**Comment by michael-radency** [comment]
> @michael-radency I was very interested, if you can create a dockerhub image I can do tests for you.

thanks @luizeof , it's already on final stages of review, should be in master soon

File: packages/nodes-base/nodes/Discord/v2/actions/router.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+	} catch (error) {}
+	const operation = this.getNodeParameter('operation', 0);
+
+	let guildId = '';
+
+	if (resource !== 'webhook') {
+		guildId = this.getNodeParameter('guildId', 0, {}, { extractValue: true }) as string;
+	}
+
+	const googleBigQuery = {
```
Comment (maspio) [inline packages/nodes-base/nodes/Discord/v2/actions/router.ts:27]: I think this part was copy/paste from google big query 😄

**Review by maspio — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.17.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.17.0)

**Pull request merged**